WooCommerce/Resources/Images.xcassets/card-reader-bbpos-chipper.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "card-reader-bbpos-chipper.svg",
```

Comment:
These images have slight borders round them which look pretty bad on dark mode. I can see there's the same on Android, but for whatever reason (maybe just because they're bigger?) they look better there.
![IMG_DC1504BCE179-1](https://user-images.githubusercontent.com/2472348/177315735-edeab25b-9662-4a80-ab90-91ac769e8940.jpeg)

Just to let you know, nothing to do right now, but one to test and watch out for.

Reply:
By increasing the image size it seems to get better (and look better overall) indeed:

<img width="328" alt="Screenshot 2022-07-06 at 09 49 04" src="https://user-images.githubusercontent.com/3812076/177457421-7f043e32-53ec-4110-95f5-c89ed7e9990e.png">

It looks like an anti-aliasing issue but doesn't make much sense as we are using `.svg` here 🤔  

In any case, as I'm already making the rows bigger I'll increase the image size as well, but I'll keep an eye on it 👍